src/tiles/list/tile-list-compact.ts

```diff
@@ -0,0 +1,218 @@
+import { css, html, LitElement, nothing } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { customElement, property } from 'lit/decorators.js';
+import { SortParam } from '@internetarchive/search-service';
+import DOMPurify from 'dompurify';
+import { TileModel } from '../../models';
+import { formatCount, NumberFormat } from '../../utils/format-count';
+import { formatDate, DateFormat } from '../../utils/format-date';
+import { accountLabel } from './account-label';
+import '../../mediatype-icon';
+
+@customElement('tile-list-compact')
+export class TileListCompact extends LitElement {
+  @property({ type: Object }) model?: TileModel;
+
+  @property({ type: String }) baseNavigationUrl?: string;
+
+  @property({ type: Number }) currentWidth?: number;
+
+  @property({ type: Number }) currentHeight?: number;
+
+  @property({ type: Object }) sortParam: SortParam | null = null;
+
+  @property({ type: Number }) mobileBreakpoint?: number;
+
+  render() {
+    return html`
+      <div id="list-line" class="${this.classSize}">
+        <div id="thumb">${this.imageTemplate}</div>
+        <div id="title">${DOMPurify.sanitize(this.model?.title ?? '')}</div>
+        <div id="creator">
+          ${this.model?.mediatype === 'account'
+            ? accountLabel(this.model?.dateAdded)
+            : DOMPurify.sanitize(this.model?.creator ?? '')}
+        </div>
+        <div id="date">${formatDate(this.date, this.formatSize)}</div>
+        <div id="icon">
+          <mediatype-icon
+            .mediatype=${this.model?.mediatype}
+            .collections=${this.model?.collections}
+            style="--iconCustomFillColor: ${ifDefined(this.collectionColor)}"
+          >
+          </mediatype-icon>
+        </div>
+        <div id="views">
+          ${formatCount(this.model?.viewCount ?? 0, this.formatSize)}
+        </div>
+      </div>
+    `;
+  }
+
+  // Only in list, not tile
+  private get collectionColor() {
+    if (this.model?.mediatype !== 'collection') {
+      return undefined;
+    }
+    return '#4666FF';
+  }
+
+  private get imageTemplate() {
+    if (!this.model?.identifier) {
+      return nothing;
+    }
+    return html` <img
+      src="${this.baseNavigationUrl}/services/img/${this.model.identifier}"
+      alt="${this.model.identifier}"
+      class="${this.model?.mediatype}"
+    />`;
+  }
+
+  /*
+   * TODO: fix field names to match model in src/collection-browser.ts
```

Comment:
There's been changes in model.ts since this comment.
This and the next two methods, among a few others are going to be DRY'ed up using a mixin class

Reply:
`sortParam?.field` is tightly coupled to the backend, search engine names.

So far, I've decided not to change this object by adding: 
`modelField` -- local name
`label` -- English name
This could be done, but likely better a different object created and passed.
Comment will be removed on decision(s) to come